cmd/cloudx/client/command_helper.go

```diff
@@ -142,29 +171,42 @@ func NewCommandHelper(ctx context.Context, opts ...CommandHelperOption) (*Comman
 		noConfirm:        false,
 		VerboseErrWriter: io.Discard,
 		Stdin:            bufio.NewReader(os.Stdin),
-		pwReader: func() ([]byte, error) {
-			return term.ReadPassword(int(os.Stdin.Fd()))
+		openBrowserHook: func(uri string) error {
```

Comment:
```suggestion
		openBrowserHook: func(uri string) {
```

Reply:
We need the error for testing unfortunately...